src/Tools/Md2Html/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.IO;
+using NDesk.Options;
+
+namespace Md2Html
+{
+    static class Program
+    {
+        static void Main(string[] args)
+        {
+            if (CheckForHelp(args))
+            {
+                // Display help and exit
+                return;
+            }
+
+            Console.WriteLine("");
+
+            while (true)
+            {
+                var line = Console.ReadLine();
```

Comment:
should we handle exceptions inside the while(true) ? if exception is thrown...it will kill the process right ?

Reply:
Yes good point. We should exit the process if anything unexpected happens with the communication.